### DISCUSSION TITLE
好奇提问，为什么翻译之后的文件会缩小？

### DISCUSSION THREAD

**Discussion by KingmirJin** [bookfere/Ebook-Translator-Calibre-Plugin#368] [General]
可能本来的文件有113M，但是翻译整合后的文件居然只有21M，有点神奇，请问是什么损失了？
![image](https://github.com/user-attachments/assets/60472bc0-3e88-44f7-b3e1-cf790d574c9e)

**Comment by bookfere** [comment]
如果书里面有很多图片，Calibre 转换格式时默认会压缩图片大小。
